src/screens/Loading/index.js

```diff
@@ -3,12 +3,12 @@ import {ActivityIndicator, Modal, StyleSheet, View} from 'react-native';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {COLORS} from '../../utils/theme';
 
-const Loading = ({visible}) => {
+const Loading = ({visible, color = COLORS.blue}) => {
   return (
     <Modal transparent visible={visible}>
       <View style={styles.container}>
         <View style={styles.content}>
-          <ActivityIndicator size="large" color={COLORS.holytosca} />
+          <ActivityIndicator size="large" color={color} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is good as it allows the color of the `ActivityIndicator` to be customizable. However, there's a potential issue if `color` prop is not provided and `COLORS.blue` does not exist in the `COLORS` object from `utils/theme`. This would cause an error at runtime. To prevent this, you could add a fallback color that is guaranteed to exist or check if the color exists in the `COLORS` object before using it.
```diff
- const Loading = ({visible, color = COLORS.blue}) => {
+ const Loading = ({visible, color = COLORS.blue || 'blue'}) => {
```
or
```diff
- const Loading = ({visible, color = COLORS.blue}) => {
+ const Loading = ({visible, color = (COLORS.blue || COLORS.defaultColor)}) => {
```
In the second example, `defaultColor` should be a color that is guaranteed to exist in your `COLORS` object.


<!-- This is an auto-generated comment by OSS CodeRabbit -->